iroh-gossip/src/net.rs

```diff
@@ -1,18 +1,14 @@
 //! Networking for the `iroh-gossip` protocol
 
-use std::{
-    collections::HashMap, fmt, future::Future, net::SocketAddr, sync::Arc, task::Poll,
-    time::Instant,
-};
-
 use anyhow::{anyhow, Context};
 use bytes::{Bytes, BytesMut};
 use futures::{stream::Stream, FutureExt};
 use genawaiter::sync::{Co, Gen};
+use iroh_net::magic_endpoint::AddrInfo as IrohInfo;
```

Comment:
I'd say remove the `as IrohInfo` and just use `AddrInfo`

Reply:
done in [afa69d0](https://github.com/n0-computer/iroh/pull/1488/commits/afa69d0c4e051eac5a6f9ff20a8fc8df8ce7f4d7)